src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcherWrapper.java

```diff
@@ -30,12 +29,6 @@ public String getComments() {
     return lobbyWatcher == null ? "" : lobbyWatcher.getComments();
   }
 
-  public void setGame(final IGame game) {
```

Comment:
This method was introduced in 156e725, but there appears to never have been a call to it since.